gvfs-helper-client.c

```diff
@@ -1,16 +1,9 @@
 #include "cache.h"
-#include "commit.h"
```

Comment:
How about using a `fixup!`-style commit message, typically generated via `git commit --fixup <commit>`? In general, I would also suggest to sign off on it, and to add some explanation as to why the original commit needs to be fixed up.

Reply:
good point.  i always forget about using that option.